migrate.go

```diff
@@ -24,13 +24,13 @@ type RedisMigrator struct {
 	migrationLogKey string
 }
 
```

Comment:
With this change I think it is time to add documentation to the function:

```suggestion

// NewRedisMigrator creates a new migrator using the given client.
//
// The given prefix is used for internal keys which the migrator maintains to manage migrations.
```